.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+
+[*.md]
+indent_size = 4
```

Comment:
What kind of editor configuration is this here? Even if this might be useful I would prefer not to side introduce here and eventually discuss along a separate PR, can you please remove?

Reply:
Yup, I'll add this in a separate PR